models/selectArticles.js

```diff
@@ -1,6 +1,6 @@
 const db = require("../db/connection");
 
-exports.selectArticles = () => {
+exports.selectArticles = (sort_by = "created_at", order = "DESC", topic) => {
   return db
     .query("ALTER TABLE articles ADD COLUMN comment_count INT DEFAULT 0;")
```

Comment:
Not sure what this alter query is doing? Adding the comment count can be done much more efficiently in one query with SQL by using a LEFT JOIN to join comments onto reviews, at the moment it looks like this is likely to break as soon as someone makes a second request once we enter production mode (it will work with testing because we tear down and rebuild the database after every request made). I'd recommend having a look at this now before going any further as it will save a lot of bother later on